pkg/client/clientset/versioned/typed/submariner/v1alpha1/fake/fake_servicediscovery.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Since we're not using the generated clientsets/informers so I think we should just remove them as it's a lot of files.

Reply:
I am writing test which used these, I will be pushing the patch soon.